daemon/tests/happy_path.rs

```diff
@@ -152,6 +154,46 @@ async fn collaboratively_close_an_open_cfd() {
     assert_next_state!(CfdState::Closed, maker, taker, order_id);
 }
 
+#[tokio::test]
+#[cfg_attr(not(feature = "fake_clock"), ignore)]
+async fn force_close_an_open_cfd() {
+    let _guard = init_tracing();
+
+    let start_time = OliviaData::example_0().announcement().expected_outcome_time
+        - daemon::SETTLEMENT_INTERVAL
+        - time::Duration::minutes(20);
+
+    let clock_handle = daemon::clock::set_fake_clock(start_time.into());
+
+    let (mut maker, mut taker, order_id) = start_from_open_cfd_state().await;
+
+    std::mem::drop(clock_handle); // No need to fake time anymore
+
+    // Taker initiates force-closing
+    taker.force_close(order_id).await;
+
+    deliver_event!(maker, taker, Event::CommitFinality(order_id));
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
```

Comment:
Can we wait for the state transition and if it does *not* happen within x time we fail? 
That would make this more efficient, because we don't always have to wait for 5 seconds. 
Would be cool if we could optimize all the sleeps away.

Reply:
due to how `next_cfd` is implemented, we can't easily optimize them away. I could have a stab at refactoring that function at some point, but I would prefer not to do it as a part of this review.
In short, testing showed that  `next_cfd` currently can return when only one of the parties is in desired state, and we want them both being in there.